test/import/import-crd.test.ts

```diff
@@ -219,3 +237,120 @@ describe('classPrefix can be used to add a prefix to all construct class names',
     classNamePrefix: 'Foo',
   });
 });
+
+describe('safe parsing', () => {
+
+  test('description is sanitized', async () => {
+
+    const crd: ManifestObjectDefinition = {
+      apiVersion: 'apiextensions.k8s.io/v1beta1',
+      kind: 'CustomResourceDefinition',
+      metadata: {
+        name: 'testMetadata',
+      },
+      spec: {
+        group: 'testGroup',
+        names: {
+          kind: 'testNameKind',
+        },
+        validation: {
+          openAPIV3Schema: {
+            description: "*/console.log('hello')/*",
+          },
+        },
+      },
+    };
+
+    await withTempFixture(crd, async (fixture: string) => {
+      const definitions = await ImportCustomResourceDefinition.match({ source: fixture });
+      expect(definitions?.length).toEqual(1);
+      const definition = definitions ? definitions[0] : undefined;
+      expect(definition?.spec?.validation?.openAPIV3Schema?.description).toEqual("_/console.log('hello')/*");
+    });
+  });
+
+  test('throws when key is illegal', async () => {
+
+    const crd: ManifestObjectDefinition = {
+      apiVersion: 'apiextensions.k8s.io/v1beta1',
+      kind: 'CustomResourceDefinition',
+      metadata: {
+        name: 'testMetadata',
+      },
+      spec: {
+        group: 'testGroup',
+        names: {
+          kind: 'testNameKind',
+        },
+        validation: {
+          openAPIV3Schema: {
+            'description': "*/console.log('hello')/*",
+            'not a word': 'value',
+          },
+        },
+      },
+    };
+
+    await withTempFixture(crd, async (fixture: string) => {
+      await expect(() => ImportCustomResourceDefinition.match({ source: fixture })).rejects.toThrow("Key 'not a word' contains non standard characters");
+    });
+
+  });
+
+  test('strips values when illegal', async () => {
+
+    const crd: ManifestObjectDefinition = {
+      apiVersion: 'apiextensions.k8s.io/v1beta1',
+      kind: 'CustomResourceDefinition',
+      metadata: {
+        name: 'testMetadata',
+      },
+      spec: {
+        group: 'its not ok to have spaces here',
+        names: {
+          kind: 'testNameKind',
+        },
+        validation: {
+          openAPIV3Schema: {
+            description: 'Its ok to have spaces here',
+          },
+        },
+      },
+    };
+
+    await withTempFixture(crd, async (fixture: string) => {
+      const definitions = await ImportCustomResourceDefinition.match({ source: fixture });
+      expect(definitions?.length).toEqual(1);
+      const definition = definitions ? definitions[0] : undefined;
```

Comment:
```suggestion
      const definition = definitions[0];
```

Reply:
Changed these tests to actually produce the generated `.ts` tests and compare snapshots.